plugins/modules/autoscaling_group.py

```diff
@@ -1219,7 +1219,7 @@ def create_autoscaling_group(connection):
             else:
                 ag["LaunchTemplate"] = launch_object["LaunchTemplate"]
         else:
-            module.fail_json_aws(e, msg="Missing LaunchConfigurationName or LaunchTemplate")
```

Comment:
Err, e is required

Reply:
e is not defined. So I shall change fail_json_aws to fail_json